corehq/apps/translations/templates/app_translations.html

```diff
@@ -52,6 +53,32 @@
                     {% crispy backup_form %}
                 </form>
             </div>
+            <div id="blacklist-trans-form" class="tab-pane">
+                <table class="table table-striped table-hover">
+                    <thead>
+                        <th class="col-sm-2">App</th>
+                        <th class="col-sm-2">Module</th>
+                        <th class="col-sm-2">Detail or List</th>
+                        <th class="col-sm-2">Field Name</th>
+                        <th class="col-sm-4">Default Display Text</th>
+                    </thead>
+                    <tbody>
+                    {% for translation in blacklisted_translations %}
+                        <tr>
+                            <td>{{ translation.app_name }}</td>
+                            <td>{{ translation.module_id }}</td>
+                            <td>{{ translation.field_type }}</td>
+                            <td>{{ translation.field_name }}</td>
+                            <td>{{ translation.display_text }}</td>
+                        </tr>
+                    {% endfor %}
+                    </tbody>
+                </table>
+                <form class="form form-horizontal" name="product" method="post">
```

Comment:
I don't think you need this - `{% crispy blacklist_form %}` ought to include the form tag, and you can use a `Fieldset` to make the legend.